ee/tabby-db/src/threads.rs

```diff
@@ -26,17 +26,21 @@ pub struct ThreadMessageDAO {
     pub code_source_id: Option<String>,
     pub attachment: Option<Json<ThreadMessageAttachment>>,
 
-    pub code_attachments: Option<Json<Vec<ThreadMessageAttachmentCode>>>,
-    pub client_code_attachments: Option<Json<Vec<ThreadMessageAttachmentClientCode>>>,
-    pub doc_attachments: Option<Json<Vec<ThreadMessageAttachmentDoc>>>,
-
+    // Deprecated since 0.25 (not removed from db yet).
+    // FIXME(meng): remove these columns from db in 0.26.
+    // pub code_attachments: Option<Json<Vec<ThreadMessageAttachmentCode>>>,
+    // pub client_code_attachments: Option<Json<Vec<ThreadMessageAttachmentClientCode>>>,
+    // pub doc_attachments: Option<Json<Vec<ThreadMessageAttachmentDoc>>>,
```

Comment:
Should insert a blank line between lines 33 and 34, as it appears that `created_at` is being deprecated.